packages/wrangler/src/dev.tsx

```diff
@@ -506,9 +507,13 @@ export async function startDev(args: ArgumentsCamelCase<DevArgs>) {
 			watcher,
 			stop: async () => {
 				devReactElement.unmount();
-				await devReactElement.waitUntilExit();
 				await watcher?.close();
-				process.exit(0);
+			},
+			fetch: async () => {
```

Comment:
I guess we'll want to pass an options object here, so folks can fake requests to paths, provide headers etc?

Reply:
probably similar to a real fetch, so similar to this https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters